inductiva/tasks/task.py

```diff
@@ -0,0 +1,147 @@
+"""Interact with a running/completed task on the Inductiva API."""
```

Comment:
This represents the task, failed or sucessfully, so it is more than just interaction.